pkg/api/router/router.go

```diff
@@ -377,6 +377,10 @@ func Router(services *services.Services) (http.Handler, error) {
 	mux.HandleFunc("PUT /api/workflows/{id}", workflows.Update)
 	mux.HandleFunc("DELETE /api/workflows/{id}", workflows.Delete)
 
+	// Generated Images
+	mux.HandleFunc("POST /api/generated/images", generatedImages.GenerateImage)
+	mux.HandleFunc("GET /api/generated/images/{id}", generatedImages.GetGeneratedImage)
```

Comment:
I don't particularly like that the DAO lives in gateway, but the routes live here. I did it this way to make the implementation simpler. That being said, having the APIs on gateway doesn't feel quite right either. Any suggestions?